javalib/src/main/scala/java/util/concurrent/ConcurrentHashMap.scala

```diff
@@ -62,7 +62,10 @@ class ConcurrentHashMap[K, V] private (initialCapacity: Int, loadFactor: Float)
     inner.clear()
 
   override def keySet(): ConcurrentHashMap.KeySetView[K, V] =
-    new ConcurrentHashMap.KeySetView[K, V](inner.keySet())
+    keySet(null.asInstanceOf[V])
```

Comment:
@sjrd is usage of `null` like this problematic if `V` is a primitive type?

Reply:
No, because it won't escape the generic environment, so it will always remain in places that erase to `Object`.

If it *did* escape into a monomorphic context, it would unbox to the zero of the target type.

Reply:
Hmm,

```scala
val map = new java.util.concurrent.ConcurrentHashMap[String,String]()
val set = map.keySet(null)
```
does not throw NPE.

Reply:
That clearly contradicts the documentation: https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.html#keySet-V-

On JDK 11, it does throw for me. If if doesn't in earlier JVM versions, we should add an `assumeTrue` for the JDK.

Reply:
Addressed in  https://github.com/scala-js/scala-js/pull/4367/commits/1e8f329d2492c2a496cbab2d68e5a0f3d67714c9